examples/src/RecordMovieComponent.tsx

```diff
@@ -28,16 +26,16 @@ const RecordMovieComponent = ({
     }, [isRecording]);
 
     const startRecording = async () => {
-        setRecordingStatus("Recording...");
+        if ("VideoEncoder" in window) {
         setOutputStatus("");
-        setIsRecording(true);
+        setIsRecording(true);}
+        else alert("VideoEncoder not supported in this browser")
```

Comment:
Is an alert the standard way in the Simularium viewer to let users know about an issue?

Would it be better to just disable the recording buttons and add a tooltip letting the user know that their browser doesn't support video recording?

Reply:
Ok, I'll add these! Was just being quick about these changes since this is just the test bed viewer, not a public facing front end implementation. I'm operating on the assumption that a dev using Firefox would have context for the limitations of this feature.

Reply:
Just added a printout instead of tooltip.